background/web3.js

```diff
@@ -17,6 +17,8 @@
 import Config, { DEFAULT_CONFIG } from './config';
 import { TRANSPORT_UNINITIALIZED } from '../shared';
 
+const FAILURE = 'Failed to fetch';
```

Comment:
The error is always like this (it's returned by Chrome this way), I don't really see an advantage of having regex here.